### PR TITLE
Refactor: Remove @helium/currency and support non-helium tokens 

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

File: android/.settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -1,11 +1,11 @@
-arguments=
+arguments=--init-script /var/folders/ck/r25tcygs77n6hv8d515p1w_c0000gn/T/d146c9752a26f79b52047fb6dc6ed385d064e120494f96f08ca63a317c41f94c.gradle --init-script /var/folders/ck/r25tcygs77n6hv8d515p1w_c0000gn/T/52cde0cfcf3e28b8b7510e992210d9614505e0911af0c190bd590d7158574963.gradle
```
Comment (abhay) [inline android/.settings/org.eclipse.buildship.core.prefs:1]: 1) 
What

Comment (ChewingGlass) [reply android/.settings/org.eclipse.buildship.core.prefs:1]: I wish I knew. React-native just did this when I installed Android. Thanks, I hate it.

File: package.json [hunk 2]
```diff
@@ -39,24 +39,25 @@
...
-    "@helium/distributor-oracle": "^0.2.6",
-    "@helium/fanout-sdk": "0.1.2",
-    "@helium/helium-entity-manager-sdk": "^0.2.6",
-    "@helium/helium-react-hooks": "0.1.2",
-    "@helium/helium-sub-daos-sdk": "0.1.2",
+    "@helium/data-credits-sdk": "0.2.16",
+    "@helium/distributor-oracle": "^0.2.16",
+    "@helium/fanout-sdk": "^0.2.16",
+    "@helium/helium-entity-manager-sdk": "^0.2.16",
+    "@helium/helium-react-hooks": "file:.yalc/@helium/helium-react-hooks",
```
Comment (abhay) [inline package.json:52]: todo?

File: android/.settings/org.eclipse.buildship.core.prefs (hunk 1 above)
Comment (ChewingGlass) [reply android/.settings/org.eclipse.buildship.core.prefs:1]: I'm just going to revert it lol

**Review by bryzettler — APPROVED** [review]
Crazy refactor, nothing major. Ill branch off of it on my bugfix and mess around with the features/ux and see if anything looks off

**Review by ThornM9 — APPROVED** [review]

**Pull request merged**